libs/langchain-azure-openai/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@langchain/azure-openai",
+  "version": "0.0.1",
```

Comment:
Just double checking, is this the intended version?

Reply:
Yes. This is consistent with other lib integration packages